doxygen particle warnings

```
/home/bob/source/include/aspect/particle/particle.h:115: warning: argument 'location' of command @param is not found in the argument list of aspec
t::Particle::Particle< dim, spacedim >::Particle(const Point< spacedim > &new_location, const Point< dim > &new_reference_location, const types::p
article_index new_id)
/home/bob/source/include/aspect/particle/particle.h:115: warning: argument 'reference_location' of command @param is not found in the argument lis
t of aspect::Particle::Particle< dim, spacedim >::Particle(const Point< spacedim > &new_location, const Point< dim > &new_reference_location, cons
t types::particle_index new_id)
/home/bob/source/include/aspect/particle/particle.h:115: warning: argument 'id' of command @param is not found in the argument list of aspect::Particle::Particle< dim, spacedim >::Particle(const Point< spacedim > &new_location, const Point< dim > &new_reference_location, const types::particle_index new_id)
/home/bob/source/include/aspect/particle/particle.h:125: warning: The following parameters of aspect::Particle::Particle::Particle(const Point< spacedim > &new_location, const Point< dim > &new_reference_location, const types::particle_index new_id) are not documented:
  parameter 'new_location'
  parameter 'new_reference_location'
  parameter 'new_id'
/home/bob/source/include/aspect/particle/particle.h:139: warning: argument 'property_pool' of command @param is not found in the argument list of aspect::Particle::Particle< dim, spacedim >::Particle(const void *&begin_data, PropertyPool &new_property_pool)
/home/bob/source/include/aspect/particle/particle.h:154: warning: The following parameters of aspect::Particle::Particle::Particle(const void *&begin_data, PropertyPool &new_property_pool) are not documented:
  parameter 'new_property_pool'
```

@gassmoeller can you take a look please?

--- a/include/aspect/particle/particle.h
+++ b/include/aspect/particle/particle.h
@@ -122,9 +122,9 @@ namespace aspect
          * in the coordinate system of the reference cell.
          * @param[in] id Globally unique ID number of particle.
          */
-        Particle (const Point<spacedim> &new_location,
-                  const Point<dim> &new_reference_location,
-                  const types::particle_index new_id);
+        Particle (const Point<spacedim> &location,
+                  const Point<dim> &reference_location,
+                  const types::particle_index id);
 
         /**
          * Copy-Constructor for Particle, creates a particle with exactly the
@@ -152,7 +152,7 @@ namespace aspect
          * by @p property_pool.
          */
         Particle (const void *&begin_data,
-                  PropertyPool &new_property_pool);
+                  PropertyPool &property_pool);
 
 #ifdef DEAL_II_WITH_CXX11
         /**

--- a/source/particle/particle.cc
+++ b/source/particle/particle.cc
@@ -37,13 +37,13 @@ namespace aspect
 
 
     template <int dim, int spacedim>
-    Particle<dim,spacedim>::Particle (const Point<spacedim> &new_location,
-                                      const Point<dim> &new_reference_location,
-                                      const types::particle_index new_id)
+    Particle<dim,spacedim>::Particle (const Point<spacedim> &location,
+                                      const Point<dim> &reference_location,
+                                      const types::particle_index id)
       :
-      location (new_location),
-      reference_location (new_reference_location),
-      id (new_id),
+      location (location),
+      reference_location (reference_location),
+      id (id),
       property_pool(NULL),
       properties (PropertyPool::invalid_handle)
     {

